app/assets/scss/_end-your-search-page.scss

```diff
@@ -0,0 +1,10 @@
+.end-search-page {
```

Comment:
Could this go in the buttons section of the front-end toolkit?

Reply:
Perhaps with a more generic name, to allow for reuse.

Reply:
We're gonna go with the standard (bigger) button, anyway :) - thanks